### PR TITLE
Liveness probe does not respect the next periodSeconds if timeoutSeconds over periodSeconds

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1956644

Previews:
Application health: https://deploy-preview-32219--osdocs.netlify.app/openshift-enterprise/latest/applications/application-health.html#application-health-about_application-health
Monitoring virtual machine health: https://deploy-preview-32219--osdocs.netlify.app/openshift-enterprise/latest/virt/logging_events_monitoring/virt-monitoring-vm-health.html
Enabling your operator: https://deploy-preview-32219--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs

### CODE REVIEW

File: modules/application-health-configuring.adoc [hunk 1]
```diff
@@ -35,42 +35,48 @@ spec:
...
-<10> Specify a host IP address. When `host` is not defined, the `PodIP` is used.
-<11> Specify `HTTP` or `HTTPS`. When `scheme` is not defined, the `HTTP` scheme is used.
-<12> Specify the port on which the container is listening.
-<13> Optional: Create a Startup probe.
-<14> Specify the type of test to perform, here an Container Execution probe.
-<15> Specify the commands to execute on the container.
-<16> Specify the number of times to try the probe after a failure.
-<17> Specify the number of seconds to perform the probe.
+<7> Specify the number of seconds to perform the probe. The default is `10`. Must be greater than `timeoutSeconds`.
+<8> Specify the number of seconds of inactivity after which the probe is assumed to have failed. The default is `1`. Must be lower than `periodSeconds`.
```
Comment (mburke5678) [inline modules/application-health-configuring.adoc:62]: <9> to <17> the text did not change, only bumped the number.
<18> Modified the text (was <17>.
<19> New text

File: modules/application-health-configuring.adoc [hunk 2]
```diff
@@ -35,42 +35,48 @@ spec:
...
+<9> Optional: Create a Readiness probe.
+<10> Specify the type of test to perform, here an HTTP test.
+<11> Specify a host IP address. When `host` is not defined, the `PodIP` is used.
+<12> Specify `HTTP` or `HTTPS`. When `scheme` is not defined, the `HTTP` scheme is used.
+<13> Specify the port on which the container is listening.
+<14> Optional: Create a Startup probe.
+<15> Specify the type of test to perform, here an Container Execution probe.
+<16> Specify the commands to execute on the container.
+<17> Specify the number of times to try the probe after a failure.
```
Comment (mburke5678) [inline modules/application-health-configuring.adoc:71]: <18> Modified the text (was <17>.
<19> New text

File: modules/application-health-about.adoc [hunk 3]
```diff
@@ -51,8 +51,8 @@ initialization is complete.
 You can configure several fields to control the behavior of a probe:
 
 * `initialDelaySeconds`: The time, in seconds, after the container starts before the probe can be scheduled. The default is 0.
-* `periodSeconds`: The delay, in seconds, between performing probes. The default is `10`.
-* `timeoutSeconds`: The number of seconds of inactivity after which the probe times out and the container is assumed to have failed. The default is `1`.
+* `periodSeconds`: The delay, in seconds, between performing probes. The default is `10`. Must be greater than `timeoutSeconds`.
```
Comment (bmcelvee) [inline modules/application-health-about.adoc:54]: This is clear as is, so feel free to ignore: 

For "Must be greater/less than..." references, would it be helpful to the user to clarify, "The value must be greater/less than..."?

**Pull request merged**